readtags.c

```diff
@@ -285,7 +288,11 @@ static int readTagLineRaw (tagFile *const file, int *err)
 		char *const pLastChar = file->line.buffer + file->line.size - 2;
 		char *line;
 
+#ifndef _WIN64
 		file->pos = ftell (file->fp);
+#else
+		file->pos = _ftelli64 (file->fp);
+#endif
```

Comment:
Instead of putting #ifndef/#else/#endif multiple places, we can introduce functions like
```
static off_t readtags_ftell(FILE *fp);
static int   readtags_fseek(FILE *fp, off_t pos, int whence);
```

We can aggregate the changes into the implementation of the two functions.

What do you think about this idea?